python/cudf/cudf/core/dataframe.py

```diff
@@ -3028,7 +3028,9 @@ def sort_index(self, ascending=True):
         """
         return self.take(self.index.argsort(ascending=ascending))
 
-    def sort_values(self, by, ascending=True, na_position="last"):
+    def sort_values(
+        self, by, ascending=True, na_position="last", ignore_index=False
```

Comment:
Can we align this API signature to the Pandas API and throw a `NotImplementedError` on any arg we don't support? https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.sort_values.html